### PR TITLE
meson: use custom install script to respect DESTDIR

### LINKED ISSUE #493: meson doesn't respect DESTDIR
**Reporter Info**

- Materia version: 8b2ee6d54e5d28240b718e7087280a00ebee460a
- GTK3 version: N/A
- Distribution (and version): Any 
- Desktop environment (and version): N/A 
- Related application (and version): N/A

**Actual Issue**

meson doesn't respect DESTDIR when installing. That breaks 100% of distribution packaging

### PR DESCRIPTION
**Pull request by maxice8**
fixes #493

### CODE REVIEW

File: scripts/install-theme.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+mv "$DESTDIR"/"$1" "$DESTDIR"/"$2"
```
Comment (actionless) [inline scripts/install-theme.sh:2]: i don't like the idea with extracting it into separate shell script

Comment (maxice8) [reply scripts/install-theme.sh:2]: I took the idea from their official documentation https://mesonbuild.com/Installing.html#custom-install-behavior

I don't personally mind how it is solved as long as it is.

**Comment by nana-4** [comment]
Thanks for filing the report and PR!

I've fixed it in a different way without adding a separate shell script, see:

**Comment by maxice8** [comment]
Thanks for fixing it

**Pull request closed without merging**